src/components/MaterialUI/DataDisplay/TimelineProps.js

```diff
@@ -0,0 +1,36 @@
+import ComponentProps from "../componentProps";
+import TimelineItemProps from "./TimelineItemProps";
+
+class TimelineProps extends ComponentProps {
+	static propNames = {
+		items: "items",
```

Comment:
"items" should not be a prop for a  TimeLine. A prop usually regards the behavior of the component, not its intrinsic nature. For example, rows are not a prop of the Table component, but its height could be,